guide/onboarding/creating-a-new-wallet.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: guide
+title: Creating a new a wallet
+description: Steps to take when creating a new wallet
+parent: Onboarding
+nav_order: 2
+permalink: /guide/onboarding/creating-a-new-wallet/
+main_classes: -no-top-padding
+---
+
+
+# What is a bitcoin wallet?
```

Comment:
There's a disconnect with the page being called `Creating a new wallet` and the first big title being `What is a bitcoin wallet?`. I'd restructure slightly to make this consistent.